package/src/types/types.ts

```diff
@@ -437,7 +437,8 @@ export enum AVEncodingOption {
   wav = 'wav',
 }
 
-export enum AVModeIOSOption {
+// TODO: Remove this enum when audio recorder player is removed completely
+export enum AVModeIOSOptionAudioRecorderPlayer {
```

Comment:
This is breaking - please keep the naming the same (and we can change it in the next major whenever we remove the other library).